src/main/java/org/broadinstitute/consent/http/authentication/OAuthAuthenticator.java

```diff
@@ -102,9 +99,15 @@ private AuthUser getUserWithStatusInfo(AuthUser authUser) {
         } else {
           logWarn("Error posting to Sam, AuthUser not able to be registered: " + gson.toJson(authUser));
         }
-      } catch (Exception exc) {
-        logException("AuthUser not able to be registered: '" + gson.toJson(authUser), exc);
+        // if post response is not successful this will be thrown, propagate the error to the user
+      } catch (WebApplicationException ex) {
+        throw ex;
+        // if building the request or parsing the response fails, this will be thrown
+        // authenticationExceptions are caught and rethrown as a 500 error in AuthFilter
+      } catch (Exception ex2) {
```

Comment:
This could also be `ex` since its scope is separate from the previous `catch`
```suggestion
      } catch (Exception ex) {
```